php/composer.json

```diff
@@ -0,0 +1,17 @@
+{
+    "name": "varias/php",
+    "type": "project",
```

Comment:
Is the idea that we install this package into our projecst (in that case the `type` here should not be project), or that we use this as a base file for projects? I would go for the first one personally (and in that case also change the `name` to something more descriptive).